### PR TITLE
Fix example 

### PR DESCRIPTION
**Pull request by willbasky**
- Add stack configuration. 
- Fix example: add parse '+' and '-' to consume Timezones +0100, -0100...

### CODE REVIEW

File: src/Titan/ArgParser.hs [hunk 1]
```diff
@@ -16,14 +16,16 @@ data Options = Options
...
+  deriving Show
 
 makeLenses ''Options
 
 parseArgs :: [String] -> IO (NS.HostName, Options)
 parseArgs args =
   case getOpt RequireOrder options args of
-    (actions, [hostname], _) ->
-      (,) <$> pure hostname <*> (foldl (>>=) (return defaultOptions) actions)
+    (actions, [hostname], _) -> do
```
Comment (willbasky) [inline src/Titan/ArgParser.hs:26]: Just for debug.

File: app/Main.hs [hunk 2]
```diff
@@ -23,45 +44,45 @@ type MyAPI = "date" :> Get Day
...
 
-instance ToResponse ZonedTime mime where
-  toResponse _ a = pack $ show a
+instance ToResponse ZonedTime where
+  toResponse a = pack $ show a
 
 instance FromHttpApiData Timezone where
   parseUrlPiece t = Timezone <$> (readTextData t :: Either Text TimeZone)
 
-handleDate :: Server mime (Get Day)
```
Comment (willbasky) [inline app/Main.hs:35]: With extra argument it not built.

**Comment by solomon-b** [comment]
Hi. Thanks for the pull request! I haven't worked on this project in a while and left it in a not ideal state. I was experimenting with using multi parameter typeclasses to dispatch based on request MIME but never finished any of that. I'll have a look through your PR in the next few days and respond with more specific comments.

**Comment by solomon-b** [comment]
So sorry about the delay in reviewing this. All of these changes look great. I especially appreciate you adding additional documentation. I'll merge it right now.

I want to start actively working on this project again. I'm going to think about what features would be nice to have and start writing issues for them. If there is anything you would like to see in this project please feel free to write up some issues describing features.

**Pull request merged**